packages/reason-expo/src/Constants.re

```diff
@@ -1,29 +1,64 @@
-[@bs.module "expo-constants"] external appOwnership: string = "appOwnership";
 
-[@bs.module "expo-constants"] external expoVersion: string = "expoVersion";
+type platforms = array(string);
 
-[@bs.module "expo-constants"]
-external installationId: string = "installationId";
+[@bs.deriving {abstract: light}]
+type ios = {supportsTablet: bool};
 
-[@bs.module "expo-constants"] external deviceId: string = "deviceId";
-
-[@bs.module "expo-constants"] external linkingUrl: string = "linkingUrl";
-
-[@bs.module "expo-constants"] external deviceName: string = "deviceName";
-
-[@bs.module "expo-constants"]
-external deviceYearClass: int = "deviceYearClass";
-
-[@bs.module "expo-constants"]
-external getWebViewUserAgentAsync: unit => Js.Promise.t(string) =
-  "getWebViewUserAgentAsync";
+[@bs.deriving {abstract: light}]
+type developer = {
+  projectRoot: string,
+  tool: string,
+};
+[@bs.deriving {abstract: light}]
```

Comment:
would be nice to space out the different type blocks

Reply:
Ran `reformat`, did some spacing. Do you want me to take out the `TOOD's`?

Reply:
those comments are todos?

Reply:
There were todos. I wonder if reformat strips that out. At any rate, the point was that those fields on the response object are unimplemented. I guess since they were unimplemented before it doesn't matter. I will take it out.

Reply:
yeah there's no listeners in the docs on constants anyway, so i was confused why those would be todos at all. it looks good now i'll merge and update NPM now. thanks for the contribution and happy coding!

Reply:
Back at you, @FiberJW. Happy to help!